client/src/Components/Login/Login.js

```diff
@@ -64,6 +75,10 @@ function Login({ className, assignCredentials }) {
         className='pic'
         alt='login femalecodercat'
       ></img>
+      <a href='http://localhost:8080/oauth/github'>
```

Comment:
I know you plan to move this to the production URL once you deploy the backend but I'd like to propose you a way I usually do it to avoid a second PR:
```suggestion
      <a href={`${process.env.API_URL}/oauth/github`}>
```